docker-selenium.yaml

```diff
@@ -173,42 +205,133 @@ subpackages:
-              mkdir -p ${{targets.subpkgdir}}/home/$SEL_USER/.vnc
-              mkdir -p ${{targets.subpkgdir}}/opt/selenium
-              mkdir -p ${{targets.subpkgdir}}/opt/selenium
-              echo "${SEL_PASSWD}" | x11vnc -storepasswd - ${{targets.subpkgdir}}/home/$SEL_USER/.vnc/passwd
-      # https://github.com/SeleniumHQ/docker-selenium/blob/trunk/NodeChrome/Dockerfile
+              cp start-selenium-node.sh ${{targets.contextdir}}/opt/bin/
+              cp start-xvfb.sh ${{targets.contextdir}}/opt/bin/
+              cp selenium.conf ${{targets.contextdir}}/etc/supervisor/conf.d/
+              cp start-vnc.sh ${{targets.contextdir}}/opt/bin/
+              cp start-novnc.sh ${{targets.contextdir}}/opt/bin/
+              cp selenium_grid_logo.png ${{targets.contextdir}}/usr/share/images/fluxbox/ubuntu-light.png
+              cp generate_relay_config ${{targets.contextdir}}/opt/bin/generate_relay_config
+
+              mkdir -p ${{targets.contextdir}}/home/$SEL_USER/.fluxbox
+              mkdir -p ${{targets.contextdir}}/tmp/.X11-unix
+              mkdir -p ${{targets.contextdir}}/home/$SEL_USER/.vnc
+              mkdir -p ${{targets.contextdir}}/opt/selenium
+              mkdir -p ${{targets.contextdir}}/opt/selenium
+              echo "${SEL_PASSWD}" | x11vnc -storepasswd - ${{targets.contextdir}}/home/$SEL_USER/.vnc/passwd
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+            - selenium-server
+            - ${{package.name}}-config
+            - geckodriver
+        environment:
+          SE_EVENT_BUS_PUBLISH_PORT: 4442
+          SE_EVENT_BUS_SUBSCRIBE_PORT: 4443
+          SE_EVENT_BUS_HOST: selenium-hub
+          CONFIG_FILE: /opt/selenium/config.toml
+          SE_BIND_HOST: True
+      pipeline:
+        - name: "Check all in one status"
+          uses: test/daemon-check-output
+          with:
+            start: "/opt/bin/start-selenium-node.sh"
+            timeout: 60
+            expected_output: |
+              Started Selenium node
+
+  # https://github.com/SeleniumHQ/docker-selenium/blob/trunk/NodeChrome/Dockerfile
+  - name: ${{package.name}}-nodechrome
+    description: NodeChrome
+    dependencies:
+      runtime:
+        - ${{package.name}}-nodebase
+        - chromium-docker-selenium-compat
+    pipeline:
       - working-directory: NodeChrome
         pipeline:
           - runs: |
-              cp wrap_chrome_binary ${{targets.subpkgdir}}/opt/bin/wrap_chrome_binary
-      # https://github.com/SeleniumHQ/docker-selenium/blob/trunk/Standalone/Dockerfile
-      - working-directory: Standalone
+              mkdir -p ${{targets.contextdir}}/opt/bin/
+              cp wrap_chrome_binary ${{targets.contextdir}}/opt/bin/wrap_chrome_binary
+    test:
+      environment:
+        contents:
+          packages:
+      pipeline:
+        - runs: |
+            /opt/bin/wrap_chrome_binary &
```

Comment:
should we use `test/daemon-check-output` here?

Reply:
yep, doing that currently. Working through all the packages